ts/Core/Navigator.ts

```diff
@@ -1744,7 +1744,13 @@ class Navigator {
         }
 
         // Update position of navigator shades, outline and handles (#12573)
-        if (navigator.navigatorEnabled) {
+        if (
+            navigator.navigatorEnabled &&
+            navigator.zoomedMin &&
+            navigator.zoomedMax
```

Comment:
When dealing with numbers always use `isNumber()` to avoid misinterpreting `zoomedMin` and `zoomedMax` value 0.

Reply:
oh right, my mistake. Thanks